site/en/r2/tutorials/eager/custom_layers.ipynb

```diff
@@ -276,13 +276,13 @@
         "    super(ResnetIdentityBlock, self).__init__(name='')\n",
         "    filters1, filters2, filters3 = filters\n",
         "\n",
-        "    self.conv2a = tf.keras.layers.Conv2D(filters1, (1, 1))\n",
+        "    self.conv2a = tf.keras.layers.Conv2D(filters1, kernel_size)\n",
```

Comment:
This is supposed to be 1x1 because its a 3 layer resnet identity block according to the resnet architecture.

Reply:
Please see the diagram here (https://www.researchgate.net/figure/Bottleneck-Blocks-for-ResNet-50-left-identity-shortcut-right-projection-shortcut_fig2_321347448) or the architecture in the paper.

Thanks but closing this PR :)

Reply:
yes, i agree but it's already the case when we use the variable kernel_size which is equals to 1

Reply:
Yes, but the kernel_size can be changed to anything, but the first and the last layer in the block should still be 1x1.

Reply:
ah, ok. you right, thanks.

Reply:
This might make a good code comment :)

Reply:
Yeah, also you can change the kernel_size parameter to 3